### ISSUE TITLE
running RwRowanProjectIssuesTest>>testIssue150_branches test gives walkback even though test passes

### ISSUE THREAD

**Issue by ericwinger** [ericwinger/Jade#248]
If you run ``RwRowanProjectIssuesTest>>testIssue150_branches`` through Jadeite>method list>Run Tests, the tests passes but you get a walkback because the test harness code expects a boolean. 

Presumably we should expect a string, too?

```
11:40:11 AM, Monday, October 22, 2018: 'Must be boolean'
String(Object)>>error:
String(Object)>>mustBeBoolean
JadeiteProjectsBrowserPresenter>>runMethodTests
Symbol>>forwardTo:
CommandDescription>>performAgainst:
[] in Command>>value
BlockClosure>>ifCurtailed:
BlockClosure>>ensure:
Command>>value
ShellView>>performCommand:
JadeiteProjectsSystemBrowser(Shell)>>performCommand:
CommandQuery>>perform
DelegatingCommandPolicy(CommandPolicy)>>route:
[] in ListView(View)>>onCommand:
BlockClosure>>ifCurtailed:
BlockClosure>>ensure:
Cursor>>showWhile:
ListView(View)>>onCommand:
ListView(View)>>wmCommand:wParam:lParam:
ListView(View)>>dispatchMessage:wParam:lParam:
[] in InputState>>wndProc:message:wParam:lParam:cookie:
BlockClosure>>ifCurtailed:
InputState>>wndProc:message:wParam:lParam:cookie:
InputState>>pumpMessage:
InputState>>loopWhile:
InputState>>mainLoop
[] in InputState>>forkMain
ExceptionHandler(ExceptionHandlerAbstract)>>markAndTry
[] in ExceptionHandler(ExceptionHandlerAbstract)>>try:
BlockClosure>>ifCurtailed:
BlockClosure>>ensure:
ExceptionHandler(ExceptionHandlerAbstract)>>try:
BlockClosure>>on:do:
[] in BlockClosure>>newProcess
```

**Comment by ericwinger** [comment]
To run tests in the browser, we use RowanAnsweringService>>runMethodTests: to return true if the tests completed successfully. 

However, that code was assuming that the only service to be returned was the answering service. This particular test returned another service. Since we don't want gui updates on server tests, we now squash other services from coming back to the client in ``RowanAnsweringService>>runMethodTests:``. 

Fixed in rowan oscar branch 3.0.43

**Comment by LisaAlmarode** [comment]
I can't actually run the specified test (see error below).  However, I can create a test that returns a project service and that is handled fine. 

a UserDefinedError occurred (error 2318), reason:halt, performOnServer: 'set -e; cd $ROWAN_PROJECTS_HOME/Rowan/test/testRepositories/repos//RowanSample3; git --git-dir $ROWAN_PROJECTS_HOME/Rowan/test/testRepositories/repos//RowanSample3/.git --work-tree $ROWAN_PROJECTS_HOME/Rowan/test/testRepositories/repos//RowanSample3 push origin issue_150' stdout: 'ERROR: Permission to dalehenrich/RowanSample3.git denied to LisaAlmarode.
fatal: Could not read from remote repository.
